.github/workflows/test.yml

```diff
@@ -1,11 +1,16 @@
 name: Tests
+
 on:
   push:
     branches-ignore:
       - "translations"
       - "dependabot/**"
   pull_request:
 
+env:
+  CI: true
```

Comment:
Already set by default https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables

Reply:
Oh, good, I guess they finally added it, because it wasn't the case in the beginning.